components/sbm-support-boot/src/test/java/org/springframework/sbm/boot/common/conditions/HasSpringBootStarterParentTest.java

```diff
@@ -0,0 +1,96 @@
+package org.springframework.sbm.boot.common.conditions;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.sbm.engine.context.ProjectContext;
+import org.springframework.sbm.project.resource.TestProjectContext;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class HasSpringBootStarterParentTest {
```

Comment:
Would a module A that has module B as parent which has spring boot as parent return true here?
Otherwise, maybe a `HasDeclaredSpringBootStarterParent` would be a more cohesive name?

Reply:
Condition checks only explicit parent. Renamed.